### PR TITLE
Add Force_Disable_Exclusive_Fullscreen_Mode to UI

### PR DESCRIPTION
**Pull request by Darkhost1999**
Fixes https://github.com/RPCS3/rpcs3/issues/11376
Still learning GitHub. Here is my 2nd ever PR.
I tested locally the setting works and defaults towards false.
Feel free to edit or suggest changes.

### CODE REVIEW

File: rpcs3/rpcs3qt/settings_dialog.ui [hunk 1]
```diff
@@ -939,6 +939,13 @@
                   <string>Asynchronous Texture Streaming</string>
                  </property>
                 </widget>
+               </item>
+			   <item>
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.ui:943]: Use spaces instead of tabs

File: rpcs3/rpcs3qt/settings_dialog.ui [hunk 2]
```diff
@@ -939,6 +939,13 @@
...
+                <widget class="QCheckBox" name="ForceDisableExclusiveFullscreenMode">
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.ui:944]: First character of the name here should be lowercase to match all the other objects

**Comment by kd-11** [comment]
Move the checkbox to the advanced tab. Messing with this option can introduce unwanted behavior especially when 3d party software is in use (OBS, etc)

File: rpcs3/rpcs3qt/tooltips.h [hunk 3]
```diff
@@ -171,7 +171,8 @@ class Tooltips : public QObject
...
 		const QString shader_compiler_threads         = tr("Number of threads to use for the shader compiler backend.\nOnly has an impact when shader mode is set to one of the asynchronous modes.");
 
-		const QString async_texture_streaming         = tr("Stream textures to GPU in parallel with 3D rendering using asynchronous compute.\nCan improve performance on more powerful GPUs that have spare headroom.\nOnly works with Vulkan renderer and greatly benefits from having MTRSX enabled if you have a capable CPU.");
+		const QString async_texture_streaming                  = tr("Stream textures to GPU in parallel with 3D rendering using asynchronous compute.\nCan improve performance on more powerful GPUs that have spare headroom.\nOnly works with Vulkan renderer and greatly benefits from having MTRSX enabled if you have a capable CPU.");
+		const QString force_disable_exclusive_fullscreen_mode  = tr("Use when you wish to stream using vulkan or if your screen goes dim using hdr.\nNote rpcs3 does not use hdr at all.");
```
Comment (kd-11) [inline rpcs3/rpcs3qt/tooltips.h:175]: Missing an explanation of what the setting actually does:
`Force borderless windowed mode for all fullscreen windows. Disables fullscreen driver optimizations but .... <add the rest here>`

**Comment by Darkhost1999** [comment]
Ok. I see I gotta click add comment before I just click resolve conversation.
I got all those changes added. Moved the setting to the Advanced tab.
Feel like I wish I knew how to bundle changes into a singular commit.
Yet setting works locally. Waiting on checks.
Any other changes?

**Comment by kd-11** [comment]
> Ok. I see I gotta click add comment before I just click resolve conversation. I got all those changes added. Moved the setting to the Advanced tab. Feel like I wish I knew how to bundle changes into a singular commit. Yet setting works locally. Waiting on checks. Any other changes?

We can squash on github before merging so don't worry about that.
Normally you would need to do an interactive rebase which allows you to squash commits.
Another option when modifying the last commit is to commit with the --amend option `git commit --amend` which just adds the new changes to the previous commit.

**Pull request merged**